frontend/Dockerfile

```diff
@@ -4,7 +4,8 @@ USER app
 WORKDIR /home/app
 
 COPY --chown=app:app ./package.json ./bun.lockb /home/app/
-RUN bun install --production --network-timeout 100000 
+# RUN bun install --production --network-timeout 100000 
+RUN bun install
```

Comment:
do we want to keep the `--production` or equivalent in the `Dockerfile`? Keep it as close to "production mode" as possible. If we need to override it, that can be done in the `docker-compose`.

Reply:
🤔think we need to include all deps since Vite is a dev dependancy but will experiment with this